### PR TITLE
feat: Replace `rtlMixin` to a composable

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR will replace the usage of `rtlMixin` to the `useUISettings` composable, and moved the method to component itself.

**Files Updated**
1. dashboard/App.vue
2. dashboard/components/widgets/conversation/ReplyBox.vue
3. dashboard/routes/dashboard/settings/reports/components/CsatTable.vue
4. dashboard/routes/dashboard/settings/reports/components/overview/AgentTable.vue
5. dashboard/routes/dashboard/contacts/components/ContactsTable.vue
6. dashboard/routes/dashboard/notifications/components/NotificationPanel.vue

Fixes https://linear.app/chatwoot/issue/CW-3471/rewrite-rtlmixin-mixin-to-a-composable

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

**Steps**
1. Navigate to account settings and set any RTL language.
2. See the UI has been updated or not.
3. And check the places where the files updated.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/composables/useUISettings.js [hunk 1]
```diff
@@ -114,6 +114,15 @@ const isEditorHotKeyEnabled = (key, uiSettings) => {
   return editorMessageKey === key;
 };
 
+/**
+ * Checks if RTL view is enabled in the UI settings.
+ * @param {Object} uiSettings - Reactive UI settings object.
+ * @returns {boolean} True if RTL view is enabled, otherwise false.
+ */
+const isRTLEnabled = uiSettings => {
+  return computed(() => uiSettings.value.rtl_view);
```
Comment (pranavrajs) [inline app/javascript/dashboard/composables/useUISettings.js:123]: @iamsivin Do we need this in the UI settings? Couldn't we just use the helper function directly in the components?

**Review by pranavrajs — CHANGES_REQUESTED** [review]
We can probably remove the composable as well. Please take a look at the comment above.

File: app/javascript/dashboard/composables/useUISettings.js (hunk 1 above)
Comment (scmmishra) [reply app/javascript/dashboard/composables/useUISettings.js]: @pranavrajs we don't need this function, it can be declared directly. I've made a commit to do this

File: app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanel.vue [hunk 2]
```diff
@@ -201,7 +196,7 @@ export default {
             <fluent-icon
               icon="chevron-left"
               size="16"
-              :class="notificationPanelFooterIconClass"
+              class="rtl:-mr-3 ltr:-ml-3"
```
Comment (scmmishra) [inline app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanel.vue:199]: @iamsivin if these `ltr` and `rtl` directives are working fine, let's use it across the app

Comment (iamsivin) [reply app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanel.vue:199]: @scmmishra We are using this `ltr` and `rtl` directives almost everywhere. It's done at the time while moved from CSS to Tailwind, Only some cases like [this](https://github.com/chatwoot/chatwoot/pull/9924/files#diff-caa95f34e5ac6da66759094fd4823106f9ee28b67477693a23a85973b7cd54c8R94) we are using it. Do you have any idea?

File: app/javascript/dashboard/composables/useUISettings.js (hunk 1 above)
Comment (pranavrajs) [reply app/javascript/dashboard/composables/useUISettings.js]: @scmmishra I don't think the change aligns with what I had in mind. Maybe I wasn't clear in the first comment. Let me try again.

I don't think this UI setting is required at all. We don't need to save it in the DB. It can be resolved dynamically from the selected language.

Instead of the following code.

```js
const { isRTL, updateUISettings } = useUISettings();
```

You could write the following. 

```js
const isRTL = getLanguageDirection(language);
```

Comment (iamsivin) [reply app/javascript/dashboard/composables/useUISettings.js]: @pranavrajs Could please you take a look? I have updated the PR https://github.com/chatwoot/chatwoot/pull/9924/commits/f959f5218f059eb8528c5ebb3af9c0b24865f39d

File: app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanel.vue (hunk 2 above)
Comment (scmmishra) [reply app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanel.vue:199]: For now it seems fine, it's out of scope for this PR atleast. We can take a look at it later

**Review by pranavrajs — APPROVED** [review]
